README.md

```diff
@@ -67,22 +73,41 @@ gcloud auth application-default login
 gcloud config set project $PROJECT_ID
 ```
 
-Initialize the Cloud infra:
+### Update Project ID
 ```
 sb set project-id $PROJECT_ID
-sb infra apply 1-bootstrap
 ```
 
-Enable Cloud Identity Platform (For Authentication)
-- https://console.cloud.google.com/marketplace/details/google-cloud-platform/customer-identity
+### Set up a Jump Host
+
+> If you choose to run this setup in your local machine, you can skip this section. However we recommend using a jump host to keep a consistant experience.
 
-Log in to the bastion host.
+Run the following to create a Compute Engine VM as the jump host.
 ```
-# TBD
+sb infra apply 0-jumphost
 ```
+- Please note that this may take 10-15 minutes to install depedencies in the VM.
 
-Set up Cloud foundation and GKE cluster
+Log into the jump host and check the depedencies setup status.
 ```
+gcloud compute ssh jump-host --zone=${ZONE} --tunnel-through-iap --project=${PROJECT_ID}
+ls -la /tmp/jumphost-ready
+```
+- If the file `jumphost-ready` exists, it means the dependencies setup is ready.
+
+Check out the code in the jump host:
+```
+git clone https://github.com/GoogleCloudPlatform/core-solution-services
+```
+
```

Comment:
We also need `gcloud auth login` and application default login steps on the jump host